scripts/build_scripts/build_zips.py

```diff
@@ -363,7 +363,7 @@ def get_windows_args():
       cmake args for windows platform.
   """
   result_args = []
-  result_args.append('-G \"Visual Studio 16 2019\"')
+  result_args.append('-G Visual Studio 16 2019')
```

Comment:
This documentation makes me feel like it should be 

```
'-G  \"Visual Studio 16 2019\"'
```

That is, it should continue to have the double quotes. No?

https://docs.microsoft.com/en-us/cpp/build/customize-cmake-settings?view=msvc-170

Reply:
But the test outcome doesn't match what the doc says